src/main/java/com/google/sps/data/DeliverySystem.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.maps.errors.ApiException;
+import com.google.sps.data.Point;
+import java.io.IOException;
+import java.lang.InterruptedException;
+import java.util.Date;
+import java.util.HashMap;
+
+public class DeliverySystem {
+  /** Variable pathFinder will be used for getting the path between two Point objects.
+   * private PathFinder pathFinder;
+   *
+   * public DeliverySystem(PathFinder pathFinder) {
+   *   this.pathFinder = pathFinder;
+   * }
+   */
```

Comment:
Please don't leave commented out code.  We can always "recover" deleted code from the change history if required - if it is not checked in yet, just store it in a future branch or in a local file until you are ready to commit it.

Reply:
I removed it.